modules/hosted-control-planes-monitoring-dashboard.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-managing.adoc
+
+:_content-type: PROCEDURE
+[id="hosted-control-planes-monitoring-dashboard_{context}"]
+= Creating monitoring dashboards for hosted clusters
+
+The HyperShift Operator can create or delete monitoring dashboards in the management cluster for each hosted cluster that it manages.
+
+//lahinson - oct. 2023 - The following paragraph was in Cesar's original draft. Does it still apply?
+
+When you enable monitoring dashboards, for each hosted cluster that the HyperShift Operator manages, the Operator creates a config map named `cp-[NAMESPACE]-[NAME]` in the `openshift-config-managed` namespace, where `NAMESPACE` is the namespace of the hosted cluster and `NAME` is the name of the hosted cluster. 
+
+[#hosted-control-planes-enable-dashboard]
+== Enabling monitoring dashboards
+
+To enable monitoring dashboards in a hosted cluster, complete the following steps:
+
+. Create the `monitoring-dashboard-template` config map in the `hypershift` namespace, being sure to specify the `--monitoring-dashboards` flag. For example:
+
++
+//lahinson - oct. 2023 - in the following example, should the name be monitoring-dashboard-template and the namespace be hypershift?
+
++
+[source,yaml]
+----
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: hypershift-operator-install-flags
+  namespace: local-cluster
+data:
+  installFlagsToAdd: "--monitoring-dashboards"
+  installFlagsToRemove: ""
+----
+
+. Log in to the management cluster's console and go to the dashboard for the hosted cluster by clicking *Observe > Dashboards*.
+
+To disable a monitoring dashboards in a hosted cluster, remove the `--monitoring-dashboards` flag from the config map. When you delete a hosted cluster, its corresponding dashboard is also deleted.
+
+[#hosted-control-planes-customize-dashboards]
+== Customizing a monitoring dashboard
+
+To generate dashboards, the HyperShift Operator uses a template that is stored in the `monitoring-dashboard-template` config map in the operator namespace (`hypershift`). This template contains a set of Grafana panels that contain the metrics for the dashboard. 
```

Comment:
At the end of this paragraph, I think we should add something like ..

You can edit the content of this ConfigMap to customize the dashboards.